README.md

```diff
@@ -1,3 +1,4 @@
-# Project A / Group 1 - Structured Data [![Join the chat at https://gitter.im/MusicConnectionMachine/StructuredData](https://badges.gitter.im/MusicConnectionMachine/StructuredData.svg)](https://gitter.im/MusicConnectionMachine/StructuredData?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Project A / Group 1 - Structured Data
 
+[![Join the chat at https://gitter.im/MusicConnectionMachine/StructuredData](https://badges.gitter.im/MusicConnectionMachine/StructuredData.svg)](https://gitter.im/MusicConnectionMachine/StructuredData?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
@kordianbruck : This comment is fixed in https://github.com/MusicConnectionMachine/StructuredData/pull/24/commits/1da39f406636b331ab954ddeacbc74407e7a263c